test/runtime/fqdn.go

```diff
@@ -866,12 +866,8 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 	})
 
 	Context("toFQDNs populates toCIDRSet when poller is disabled (data from proxy)", func() {
-		var config = `
-PATH=/usr/lib/llvm-3.8/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug --pprof=true --log-system-load --tofqdns-enable-poller=false
-INITSYSTEM=SYSTEMD`
 		BeforeAll(func() {
-			vm.SetUpCiliumWithOptions(config)
+			vm.SetUpCiliumWithNoDNSPoller()
```

Comment:
I would not move this to the helpers. It's easy to know what is doing here if we have all in the test.

Reply:
Fair point.

What I noticed was just that we have three different places where we're specifying full Cilium commandline opts, which means if we change defaults in future then we need to go through each of these and understand what the differences need to be to make a change. IMHO it'd be easier if the three locations applied the changes like "config := defaultConfig + "--to-fqdns-enable-poller=false".

But tidying that up can be subject to a separate PR, I'll remove this refactoring from here.